src/command-line/add.js

```diff
@@ -1,14 +1,21 @@
 "use strict";
 
-var ClientManager = new require("../clientManager");
-var colors = require("colors/safe");
-var program = require("commander");
-var Helper = require("../helper");
+const colors = require("colors/safe");
+const program = require("commander");
+const fs = require("fs");
+const Helper = require("../helper");
 
 program
 	.command("add <name>")
 	.description("Add a new user")
 	.action(function(name) {
+		if (!fs.existsSync(Helper.USERS_PATH)) {
+			log.error(`${Helper.USERS_PATH} does not exist.`);
+			return;
+		}
```

Comment:
It's a bit of an awkward workflow now, because to start in private mode, you need at least one user, and to add a user, you need a configuration file / tree, which gets created with `lounge start`.
It's awkward currently too, because you still need to set to private mode _after_ creating a user (creating a user should fail if mode is public, IMO, but that's a different story) but a bit less.

So the workflow after this PR will be:

1. `lounge start` to create config _and_ start a public instance
2. Kill it
3. Update configuration file to set to private mode
4. `lounge add <user>`
5. `lounge start` to start private mode with that user

This is what I think it makes a lot of sense to have a `lounge init` that would replace step 1/2 (and 3 if making it interactive, see #773).

What do you think? How can we easily change this to make it a bit less awkward?